src/ddscxx/include/dds/core/policy/CorePolicy.hpp

```diff
@@ -195,6 +198,7 @@ OMG_DDS_POLICY_TRAITS(DataRepresentation,   23)
 OMG_DDS_POLICY_TRAITS(TypeConsistencyEnforcement, 24)
 #endif  // OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT
 OMG_DDS_POLICY_TRAITS(WriterBatching,       25)
+OMG_DDS_POLICY_TRAITS(PSMXInstances,        34)
```

Comment:
Maybe this comment should be on the CycloneDDS-C repository's PR, but are we sure that this new Policy-ID will not conflict with other vendors' stuff?